### PR TITLE
"stats.js" -> "stats-js"

### PR DESCRIPTION
**Pull request by kobatakuJP**
this code will fix issue: #15696.

npm package name is "stats-js".
Therefore, 
`import Stats from "stats.js";` IDE:"yes!" Compiler:"No."
`import Stats from "stats-js";` IDE:"No." Compiler:"Yes!"
So this request. but... please tell me which is good manner from follow. I do not know good manner about this case.

1. marge it. in future, user do 
   `npm install --save stats-js`
   `npm install --save-dev @types/stats.js`
   `import Stats from "stats-js"`. 
   It is not clear for new people.
1. Do no marge it, and create `types/stats-js`. in future, user do 
   `npm install --save stats-js`
   `npm install --save-dev @types/stats-js`
    `import Stats from "stats-js"`. 
    It is clear for new people but old people get confused. "which is correct?"

### CODE REVIEW

**Review by danvk — APPROVED** [review]
This seems fine, thanks for fixing this @kobatakuJP.

I believe the directory name will also need to change, `types/stats.js` → `types/stats-js`.

My question looking at this was "how did this ever work?" At least for me, it's because I was using it in a [context](https://github.com/tt-acm/DesignExplorer/blob/0326c30d1669d52f5541a8ab3ed3053bc185086a/index.html#L90) where stats.js was sourced via a `<script>` tag, so it looks like it should be called `stats.js`.

**Comment by PranavSenthilnathan** [comment]
So there is an existing npm package called stats-js so I don't think you want to change the name of the module here. Could you share the error that you're getting from the compiler for this:
  `import Stats from "stats.js";` IDE:"yes!" Compiler:"No."

File: types/stats.js/index.d.ts [hunk 1]
```diff
@@ -29,6 +29,6 @@ declare namespace Stats {
     }
 }
 
-declare module 'stats.js' {
+declare module 'stats-js' {
```
Comment (Jessidhia) [inline types/stats.js/index.d.ts]: The folder should be (at least) renamed to `stats-js`.

`stats-js` is also now exported as a UMD bundle so this definition should be refactored like a commonjs definition (`export =` but at the top level, without `declare module`s) with an additional `export as namespace Stats`, but that could be done separately.

**Review by Jessidhia — CHANGES_REQUESTED** [review]

**Comment by Jessidhia** [comment]
It actually looks like `stats-js` is a fork of the original `stats.js` 🤔

File: types/stats.js/index.d.ts (hunk 1 above)
Comment (kobatakuJP) [reply types/stats.js/index.d.ts]: @Jessidhia 
rename folder and so on. Could you re-review?

**Comment by PranavSenthilnathan** [comment]
I'm looking at the repo and it seems like in your package.json you're adding a dependency to `@types\stats.js` and `stats-js`. I'm pretty sure if you change the `stats-js` dependency to `stats.js` as well then it should work. If you want types for stats-js then you should create a new definition file instead of touching the definition file for stats.js.

**Comment by kobatakuJP** [comment]
@PranavSenthilnathan 
Ahh.... yes, that is certainly true.
I try `npm uninstall -S stats-js` and `npm install -S stats.js` and `npm run build-d` at the repository, and it is work.
Thank you for your pointing out.

[stats.js](https://www.npmjs.com/package/stats.js) IS original, [stats-js](https://www.npmjs.com/package/stats-js) is forked. I had a misunderstanding...

So, this requests should not be merged... 

@Jessidhia @danvk 
I close this request. I'm sorry for giving you mixed signals.

**Pull request closed without merging**